builds/assets/lang/zhCN.json

```diff
@@ -9,6 +9,8 @@
     "targetVersion": "0.5.0"
   },
 
+  "popupClose": "关闭",
```

Comment:
This will probably need to be in the english translation if used also

Reply:
There is English already.